Modules/CPV/src/PhysicsTask.cxx

```diff
@@ -90,10 +88,21 @@ void PhysicsTask::monitorData(o2::framework::ProcessingContext& ctx)
   int nValidInputs = ctx.inputs().countValidInputs();
   mHist1D[H1DNValidInputs]->Fill(nValidInputs);
 
+  bool hasClusters = false, hasDigits = false;
   for (auto&& input : ctx.inputs()) {
     // get message header
     if (input.header != nullptr && input.payload != nullptr) {
       const auto* header = header::get<header::DataHeader*>(input.header);
+      //LOG(info) << "monitorData() : obtained input " << header->dataOrigin.str << "/" << header->dataDescription.str;
+      if ((strcmp(header->dataOrigin.str, "CPV") == 0) && (strcmp(header->dataDescription.str, "DIGITS") == 0)) {
+        LOG(info) << "monitorData() : I found digits in inputs";
```

Comment:
I would be extremely careful about printing and logging in `monitorData`. This is typically called at high frequency. It will create a flood. I would advise
2. to remove it, if not
2. to make the log DEBUG and limit the number of times it can be printed, and
3. to use the infologger instead of the fairlogger (not mandatory).

Reply:
Hi @Barthelemy 
indeed logging there was unnecessary and was for debug purpose. I forgot to remove it in hurry. Thanks!